<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>739</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>285</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>285</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="cameraPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>285</width>
         <height>100</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>285</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>285</width>
         <height>100</height>
        </size>
       </property>
       <attribute name="label">
        <string>相机列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="cameraComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="cameraLayout">
          <item>
           <widget class="QPushButton" name="searchCameraButton">
            <property name="text">
             <string>搜索相机</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cameraButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>打开相机</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="capturePage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>285</width>
         <height>400</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>285</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>285</width>
         <height>400</height>
        </size>
       </property>
       <attribute name="label">
        <string>捕获与分辨率</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="selectLayout">
          <item>
           <widget class="QPushButton" name="captureButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>捕获</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="videoButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>录像</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <widget class="QComboBox" name="formatComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="previewComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="previewLabel">
            <property name="text">
             <string>预览：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="captureLabel">
            <property name="text">
             <string>捕获：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="captureComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="formatLabel">
            <property name="text">
             <string>格式：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="previewSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <spacer name="captureSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="formatSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="modeLayout">
          <item>
           <widget class="QRadioButton" name="videoRadioButton">
            <property name="text">
             <string>视频模式</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="triggerRadioButton">
            <property name="text">
             <string>触发模式</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="autoExposureCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>自动曝光</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="exposureTargetLayout">
          <item>
           <widget class="QLabel" name="exposureTargetLabel">
            <property name="text">
             <string>曝光目标：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="exposureTargetSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="exposureTargetNumLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="exposureTargetSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="exposureTimeLayout">
          <item>
           <widget class="QLabel" name="exposureTimeLabel">
            <property name="text">
             <string>曝光时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="exposureTimeSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="exposureTimeNumLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="exposureTimeSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="gainLayout">
          <item>
           <widget class="QLabel" name="gainLabel">
            <property name="text">
             <string>增益：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="gainSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="gainNumLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="gainSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="whileBalancePage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>285</width>
         <height>230</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>285</width>
         <height>210</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>285</width>
         <height>230</height>
        </size>
       </property>
       <attribute name="label">
        <string>白平衡</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="temperatureLayout">
          <item>
           <widget class="QLabel" name="temperatureLabel">
            <property name="text">
             <string>色温：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="temperatureSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="temperatureNumLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="temperatureSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="tintLayout">
          <item>
           <widget class="QLabel" name="tintLabel">
            <property name="text">
             <string>Tint：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="tintSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="tintNumLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="tintSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="wbRemindLabel1">
          <property name="text">
           <string>在点击“白平衡”按钮之前，请移动ROI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="wbRemindLabel2">
          <property name="text">
           <string>(标记为红色矩形）至样品的纯白或者</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="wbRemindLabel3">
          <property name="text">
           <string>灰色部分。</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="wbSelectLayout">
          <item>
           <widget class="QPushButton" name="whileBalanceButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>白平衡</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="defaultValueButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>默认值</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="tabLayout" stretch="1,0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="serialWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="serialGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>串口通信</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="serialLayout" stretch="0,0">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="portLayout">
              <item>
               <widget class="QLabel" name="portLabel">
                <property name="minimumSize">
                 <size>
                  <width>135</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>135</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>串口号：</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="portBox">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="serialbtnLayout">
              <item>
               <widget class="QPushButton" name="openSerialButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>打开串口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="refreshPortButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>刷新串口</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="speedGroupBox">
         <property name="title">
          <string>速度控制</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="highSpeedRadioButton">
            <property name="text">
             <string>高速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mediumSpeedRadioButton">
            <property name="text">
             <string>中速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lowSpeedRadioButton">
            <property name="text">
             <string>低速</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>操作控制</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="rAxisLayout">
            <item>
             <widget class="QLabel" name="rAxisLabel">
              <property name="text">
               <string>R轴</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rAxisForwardButton">
              <property name="text">
               <string>进</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rAxisBackwardButton">
              <property name="text">
               <string>退</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="tAxisLayout">
            <item>
             <widget class="QLabel" name="tAxisLabel">
              <property name="text">
               <string>T轴</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tAxisForwardButton">
              <property name="text">
               <string>进</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tAxisBackwardButton">
              <property name="text">
               <string>退</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="messageGroupBox">
         <property name="title">
          <string>消息窗口</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QTextEdit" name="serialMessageEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSerial"/>
  </widget>
  <action name="actionSerial">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/images/control.png</normaloff>:/images/images/control.png</iconset>
   </property>
   <property name="text">
    <string>Serial</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
